tests/epics/adcore/test_writers.py

```diff
@@ -99,13 +136,14 @@ async def test_hdf_writer_file_not_found(hdf_writer: adcore.ADHDFWriter):
 
 async def test_hdf_writer_collect_stream_docs(hdf_writer: adcore.ADHDFWriter):
     assert hdf_writer._composer is None
-
+    _ = await hdf_writer.open(exposures_per_event=1)
```

Comment:
Don't need the return value here as it's None
```suggestion
    await hdf_writer.open(exposures_per_event=1)
```